packages/protocol/index.js

```diff
@@ -1,20 +1,29 @@
 const AirnodeArtifact = require('./artifacts/contracts/Airnode.sol/Airnode.json');
 const ConvenienceArtifact = require('./artifacts/contracts/Convenience.sol/Convenience.json');
-
-const networkChainIds = { ropsten: 3, rinkeby: 4 };
+const MockAirnodeClientArtifact = require('./artifacts/contracts/mock/MockAirnodeClient.sol/MockAirnodeClient.json');
 
 const AirnodeAddresses = {};
 const ConvenienceAddresses = {};
-for (const network in networkChainIds) {
-  const AirnodeDeployment = require(`./deployments/${network}/Airnode.json`);
-  AirnodeAddresses[networkChainIds[network]] = AirnodeDeployment.receipt.contractAddress;
-  const ConvenienceDeployment = require(`./deployments/${network}/Convenience.json`);
-  ConvenienceAddresses[networkChainIds[network]] = ConvenienceDeployment.receipt.contractAddress;
-}
+let AirnodeDeployment, ConvenienceDeployment;
```

Comment:
Having these mutable variables that get overwritten feels like a weird pattern to me. It may even cause unexpected behaviour if requires aren't executed in order. Last I checked, using `import` syntax doesn't respect ordering.

I would rather have unique names for each require just to be safe. Something like:

```ts
const RopstenAirnode = require('./deployments/ropsten/Airnode.json');
const RopstenConvenience = require('./deployments/ropsten/Convenience.json');

const RinkebyAirnode = require('./deployments/rinkeby/Airnode.json');
const RinkebyConvenience = require('./deployments/rinkeby/Convenience.json');
```